docs/proposals/rollout/rollout.md

```diff
@@ -288,15 +297,19 @@ type TrafficRoutingConfig struct {
     // +optional
     CorsPolicy *istiov1alpha3.CorsPolicy `json:"corsPolicy,omitempty"`
 
-    // RolloutAnalysisCheckTimes defines the number of traffic analysis checks to run for A/B Testing and Blue/Green
-    // If set "rolloutAnalysisCheckTimes: 10". It means Kurator will checks the preview service ten times. 
-    // Note: Kurator determines whether Blue/Green and A/B or canary related processing is required based on 
-    // the presence or absence of content in the rolloutAnalysisCheckTimes field. 
-    // So can't configure rolloutAnalysisCheckTimes and canaryStrategy at the same time.
+    // CanaryStrategy defines parameters for canary test.
+    // Note: Kurator determines whether A/B Testing and Blue/Green Deployment or Canary Deployment 
+    // related processing is required based on  the presence or absence of content in the analysisTimes field.
```

Comment:
please reword this sentence. can you try  totranslate into Chinese using dictionary